Dockerfile

```diff
@@ -60,21 +60,24 @@ RUN useradd --create-home --shell /bin/bash critiquebrainz
 
 # Consul Template service is already set up with the base image.
 # Just need to copy the configuration.
-COPY ./docker/prod/consul-template.conf /etc/consul-template.conf
+COPY ./docker/consul-template.conf /etc/consul-template.conf
+
+# runit service files
+# All services are created with a `down` file, preventing them from starting
+# rc.local removes the down file for the specific service we want to run in a container
+# http://smarden.org/runit/runsv.8.html
 
 COPY ./docker/$DEPLOY_ENV/uwsgi/uwsgi.service /etc/service/uwsgi/run
 RUN chmod 755 /etc/service/uwsgi/run
 COPY ./docker/$DEPLOY_ENV/uwsgi/uwsgi.ini /etc/uwsgi/uwsgi.ini
+RUN touch /etc/service/uswgi/down
 
 # cron jobs
 ADD ./docker/prod/cron/jobs /tmp/crontab
 RUN chmod 0644 /tmp/crontab && crontab -u critiquebrainz /tmp/crontab
 RUN rm /tmp/crontab
 RUN touch /var/log/dump_backup.log /var/log/public_dump_create.log /var/log/json_dump_create.log \
     && chown critiquebrainz:critiquebrainz /var/log/dump_backup.log /var/log/public_dump_create.log /var/log/json_dump_create.log
-
-# Make sure the cron service doesn't start automagically
-# http://smarden.org/runit/runsv.8.html
```

Comment:
cron is now a bit more complex, we have to do two things:

 - add down to /etc/service/cron/down
 - add a new service (call it cron-config), which calls `run-consul-template` and generates the config files necessary for cron. This isn't in LB yet.
 - When a cron container starts up, we'll activate both of these services. One to make the config and another to start cron (we don't combine them together because we don't want consul to restart cron if it's in the middle of a task)